packages/bundle-size/src/utils/collectLocalReport.js

```diff
@@ -46,7 +46,7 @@ async function readReportForPackage(reportFile) {
  */
 async function collectLocalReport() {
   /** @type {string[]} */
-  const reportFiles = glob.sync('packages/*/dist/bundle-size/bundle-size.json', {
+  const reportFiles = glob.sync('packages/**/dist/bundle-size/bundle-size.json', {
```

Comment:
This glob is less efficient, but as this package used in other repos it's not a good idea to include there `react-components`.